packages/commonwealth/client/scripts/views/pages/discussions/DiscussionsPage.tsx

```diff
@@ -84,9 +84,9 @@ const DiscussionsPage = ({ topicName }: DiscussionsPageProps) => {
         queryType: 'bulk',
         limit: 20,
         topicId,
-        stage: stageName,
+        stage: stageName ?? undefined,
```

Comment:
The schema doesn't support passing in null params now, so we need to pass undefined